### PR TITLE
Feat/Register Login Screen

### PR DESCRIPTION
**Pull request by ghost**
> A simple description of your changes

### CODE REVIEW

File: src/components/LoginForm.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { useState } from "react";
-import { Redirect } from "react-router-dom";
-import { useForm } from "../hooks";
+import {useState} from "react";
```
Comment (itsabdessalam) [inline src/components/LoginForm.jsx]: The code formatting doesn't seem to be the same anymore

File: src/components/RegisterForm.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  const register = async () => {
+    setIsLoading(true);
+    setError(null);
+
+    const user = {
+      firstName: values.firstName,
+      lastName: values.lastName,
+      email: values.email,
+      password: values.password,
+      language: navigator.language.slice(0, 2)
```
Comment (itsabdessalam) [inline src/components/RegisterForm.jsx]: Why not put it in a utils function?

File: src/components/RegisterForm.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      setIsLoading(false);
+    }
+  };
+
+  const {values, handleChange, handleSubmit} = useForm(register, {
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    language: navigator.language.slice(0, 2)
```
Comment (itsabdessalam) [inline src/components/RegisterForm.jsx]: This one doesn't have to be filled, you can simply keep it empty
```suggestion
    language: ""
```

File: src/components/index.js [hunk 4]
```diff
@@ -9,9 +10,12 @@ import Img from "./Img";
 import Input from "./Input";
 import LoginForm from "./LoginForm";
 import ProtectedRoute from "./ProtectedRoute";
+import RegisterForm from "./RegisterForm";
+import LoginRegisterBtn from "./LoginRegisterBtn";
```
Comment (itsabdessalam) [inline src/components/index.js]: Can we give it a more generic name?

File: src/components/RegisterForm.jsx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+import Form from "./Form";
+import Input from "./Input";
+import Button from "./Button";
+import { useForm } from "../hooks";
+import { useState } from "react";
+import until from "../utils/until";
+import AuthService from "../services/AuthService";
+import { handleLogin, isLoggedIn } from "../utils/auth";
+import { Redirect } from "react-router-dom";
+import getLanguage from "../utils/langage";
```
Comment (itsabdessalam) [inline src/components/RegisterForm.jsx]: Language?

File: src/utils/langage.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Get navigator langage and format then
```
Comment (itsabdessalam) [inline src/utils/langage.js]: Language?

**Review by codebynao — APPROVED** [review]

**Pull request merged**